create/index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <script defer src="https://cdn.jsdelivr.net/npm/@supabase/supabase-js"></script>
+        <link rel="stylesheet" href="../styles/reset.css" />
+        <link rel="stylesheet" href="../styles/style.css" />
+        <link rel="stylesheet" href="../styles/home.css" />
+        <title>Document</title>
+    </head>
+    <body>
+        <header>
+            <div class="gutter-right logo">
+                <a href="../workshop/">See all workshops.</a>
+                <a href="../create">Join a workshop.</a>
```

Comment:
Very minor detail, but if we're on the create page consider not having the link to create.